README.md

```diff
@@ -322,6 +322,10 @@ server {
         proxy_read_timeout 24h;
         proxy_http_version 1.1;
         proxy_set_header Connection "";
+        ## Remove comments only if you use argument USE_FORWARDED_HEADERS=1 ##
+        #proxy_set_header Host $host;
+        #proxy_set_header X-Real-IP $remote_addr;
```

Comment:
Setting this one is issueless is X-Forwarded-For is set.
However both X-Forwarded-Scheme must be set to prevent an attacker to hijack it.

Reply:
I withdraw that. Thanks for the feedback